docs/sources/developers/plugins/_index.md

```diff
@@ -6,77 +6,48 @@ title: Build a plugin
 weight: 200
 ---
 
-# Build a plugin
+# Welcome to the Grafana plugin developer's guide
 
-For more information on the types of plugins you can build, refer to the [Plugin Overview]({{< relref "../../administration/plugin-management/" >}}).
+Grafana is an open source software project for creating, exploring, and sharing all of your data through beautiful, flexible dashboards. Plugins extend its built-in functionality so that users can accomplish specialized tasks to make Grafana even more useful for them. 
```

Comment:
```suggestion
Grafana is an open source software project for creating, exploring, and sharing data through dashboards. Plugins extend Grafana's built-in functionality so that users can accomplish specialized tasks. 
```

Reply:
nice suggestions, I will check in fix